### PR TITLE
added react select

### PR DESCRIPTION
**Pull request by BilalMaqsoodA**

### CODE REVIEW

File: src/components/dashboard/riders-list/riders-sorter/riders-sorter.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ import { Select } from '../../../ui/select/select';
 import { Text } from '../../../ui/text';
 import { Styled } from '../../styled';
 
+const options = [
+  { value: 'available', label: 'available' },
```
Comment (Taimoormk) [inline src/components/dashboard/riders-list/riders-sorter/riders-sorter.js]: How are you handling the id of each hash within the array? Shouldn't there is also a key for id?

File: src/components/ui/__stories__/select.story.js [hunk 2]
```diff
@@ -1,28 +1,21 @@
...
 
 
 const stories = storiesOf('Select', module);
 
 const props = () => ({
-  id: text('id', 'userselect'),
-  name: text('name', 'userselect'),
-  values: [{ name: 'Thailand' }, { name: 'Pakistan'}],
+  options: [
+    { value: 'available', label: 'available' },
```
Comment (Taimoormk) [inline src/components/ui/__stories__/select.story.js]: Same here, can we have id for each item in the array?

File: src/components/dashboard/riders-list/riders-sorter/riders-sorter.js [hunk 3]
```diff
@@ -3,12 +3,17 @@ import { Select } from '../../../ui/select/select';
...
+  { value: 'busy', label: 'busy' },
+  { value: 'On leave', label: 'on leave' },
+];
+
 export const RidesSorter = () => (
   < Styled>
```
Comment (Taimoormk) [inline src/components/dashboard/riders-list/riders-sorter/riders-sorter.js]: Remove extra space from `< Styled>`

File: src/components/dashboard/riders-list/riders-sorter/riders-sorter.js [hunk 4]
```diff
@@ -3,12 +3,17 @@ import { Select } from '../../../ui/select/select';
...
     <Text >Sort by</Text>
```
Comment (Taimoormk) [inline src/components/dashboard/riders-list/riders-sorter/riders-sorter.js]: Extra space here too `<Text >`

File: src/components/dashboard/riders-list/riders-sorter/riders-sorter.js [hunk 5]
```diff
@@ -3,12 +3,17 @@ import { Select } from '../../../ui/select/select';
 import { Text } from '../../../ui/text';
 import { Styled } from '../../styled';
 
+const options = [
```
Comment (InamTaj) [inline src/components/dashboard/riders-list/riders-sorter/riders-sorter.js]: please move these options to selector of riders-list

File: src/components/ui/__stories__/select.story.js [hunk 6]
```diff
@@ -1,28 +1,21 @@
...
 import { Select } from '../select';
 
 
 const stories = storiesOf('Select', module);
 
 const props = () => ({
-  id: text('id', 'userselect'),
-  name: text('name', 'userselect'),
-  values: [{ name: 'Thailand' }, { name: 'Pakistan'}],
+  options: [
```
Comment (InamTaj) [inline src/components/ui/__stories__/select.story.js]: just call the selector to get the list here

File: package.json [hunk 7]
```diff
@@ -32,6 +32,7 @@
     "react-router": "^5.0.1",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.1.1",
+    "react-select": "^3.0.4",
```
Comment (InamTaj) [inline package.json]: Is there any specific reason to use this lib? Why aren't we creating our own select component?

Comment (BilalMaqsoodA) [reply package.json]: According to tanveer, it is more convenient to style react select as it is containing the markup of  button.

**Pull request merged**